sakai-evaluation-tool/src/webapp/content/templates/modify_hierarchy_node_groups.html

```diff
@@ -49,6 +49,14 @@ <h3 rsf:id="assign-groups-title">Assign groups for a hierarchy node</h3>
     </p>
     -->
     <form rsf:id="assign-groups-form">
+
+        <div class="action" style="margin-bottom: 5px;">
```

Comment:
Same as above comment; should likely be addressed in a separate JIRA project wide.